3_Development/git.md

```diff
@@ -13,40 +13,68 @@ The commit message should look like this:
+
+Further paragraphs come after blank lines.
+
+It is critical that you explain *why* you did the change. Don't state
+the obvious "File updated", "Text removed". Imagine you are reading
+this message after two years; how would you explain your future self
+the reason for this change?
+
+  - Bullet points are okay, too
+
+  * Typically a hyphen or asterisk is used for the bullet,
+    preceded by a single space, with blank lines in
+    between.
+
+At the bottom of the commit message, reference the Issue(s) to which
+this commit belongs, in the format below. If referencing an Issue in the
+repo that this commit is in, it's fine to just use the issue number.
+Otherwise, use the long format with organization/repo prefix.
+
+Refs: #123.
+Refs: niteoweb/operations#123.
+Refs:
+  - #134
+  - niteoweb/operations#145
+
+
+Bonus karma points for reading [this detailed essay about good commits](https://chris.beams.io/posts/git-commit/).
+
+A few examples of excellent commits:
+> Ignore newly created blogs in CheckBackups
 >
-> In big repos, you can use a `[tag]` prefix in the summary, if you
-> wish.
->
-> The summary is all about "What does this commit do?" while the body is
-> "Why does it do this?" and "Why was this approach chosen over others?"
->
-> Further paragraphs come after blank lines.
+> Blogs that are not older than 1 day should not be flagged by `Check Backups`
+> check.
 >
-> It is critical that you explain *why* you did the change. Don't state
-> the obvious "File updated", "Text removed". Imagine you are reading
-> this message after two years; how would you explain your future self
-> the reason for this change?
+> Ref: https://github.com/niteoweb/ebn/issues/3489
+
+> Create a GitHub issue when Provision DNS fails.
 >
->   - Bullet points are okay, too
+> When Provision DNS fails create a GitHub issue, label it as "Blog"
+> and assign it to niko (@nkk0) and adam (@by-cx).
 >
->   * Typically a hyphen or asterisk is used for the bullet,
->     preceded by a single space, with blank lines in
->     between.
+> Ref: https://github.com/niteoweb/ebn/issues/3788
+
+> Fix remaining flake8 errors
 >
-> At the bottom of the commit message, reference the Issue(s) to which
-> this commit belongs, in the format below. If referencing an Issue in the
-> repo that this commit is in, it's fine to just use the issue number.
-> Otherwise, use the long format with organization/repo prefix.
+> * Ignore C901 errors for particular function definitions.
+> * Ignore B001 errors in certain files
+> * Fix remaining E501 errors.
+> * Include `lint` command in `make tests`.
+> * Add changes from end-of-file-fixer to `pyramid_bimt` static files
+> and ignore C812 errors (missing trailing comma) that produced by `black`.
+> * Ignore E501 so it will not break line-length limit.
 >
-> Refs #123.
-> Refs niteoweb/operations#123.
-
-Bonus karma points for reading [this detailed essay about good commits](https://chris.beams.io/posts/git-commit/).
+> Ref: #3612
```

Comment:
Add complete ref link: https://github.com/niteoweb/ebn/issues/3612

Reply:
`Ref: 3612` is a valid reference. This is how the original commit looks like.